client/core/types.go

```diff
@@ -316,7 +316,7 @@ func newDisplayIDFromSymbols(base, quote string) string {
 type MiniOrder struct {
 	Qty   float64 `json:"qty"`
 	Rate  float64 `json:"rate"`
-	Epoch *uint64 `json:"epoch,omitempty"`
+	Epoch uint64  `json:"epoch,omitempty"`
```

Comment:
@JoeGruffins can check me here, but I didn't see a good reason this couldn't just be a value. `omitempty` still works as expected zero value.